pandas/tests/extension/decimal/array.py

```diff
@@ -102,7 +102,7 @@ def copy(self, deep=False):
     def astype(self, dtype, copy=True):
         if isinstance(dtype, type(self.dtype)):
             return type(self)(self._data, context=dtype.context)
-        return super(DecimalArray, self).astype(dtype, copy)
+        return np.asarray(self, dtype=dtype)
```

Comment:
OT: how much testing do we have on this?

Reply:
Which bit specifically? We have a base test for `astype` to object and series.

I'm not actually sure why I changed this...